source/configuration/read.txt

```diff
@@ -282,8 +267,8 @@ This partitioner is not compatible with hashed shard keys.
      - Description
```

Comment:
Add full name / path for all the partitioners:

`com.mongodb.spark.sql.connector.read.partitioner.PaginateIntoPartitionsPartitioner`